### ISSUE TITLE
Double click select a part of a var

### ISSUE THREAD

**Issue by djondi** [adobe/brackets#10936]
This one is simple. Double clicking on a variable "$var" selects only the "var" without the $ sign :)

**Comment by MarcelGerber** [comment]
What language is your file written in?
It should already work with JavaScript, PHP and Perl files. If it doesn't, what version of Brackets are you using? See `Help > About Brackets`.

**Comment by djondi** [comment]
I'm writing in PHP. I'm using version 1.2 build 1.2.0-15697. Believe that's the latest one.
Here's a screenshot just so we can understand each other better :)
https://www.dropbox.com/s/zf6dbljzqkdkbp6/Screenshot%202015-04-19%2008.40.07.png?dl=0

**Comment by MarcelGerber** [comment]
Yeah, I get what you want (and that's also the expected behavior). For me, it definitely select `$komentar` rather than just `komentar`.
Could you just try `Debug > Reload without extensions` to make sure it's not caused by an extension?

**Comment by djondi** [comment]
I did try to reload without any extension and it didn't help. The thing that bothers me is still there. It's just I've spent many years with other text editors (Notepad++, Sublime Text etc) and they all did select only the text without the variable sign. It's hard to start doing different things now. Is there any extension that could help me?

**Comment by MarcelGerber** [comment]
Ah, from your description I thought you meant it the other way round.

You could write a very simple extension to fix that. This should suffice:
``` js
define(function (require, exports, module) {
    var CodeMirror = brackets.getModule("thirdparty/CodeMirror2/lib/codemirror");
    CodeMirror.registerHelper("wordChars", "php", /\w/
});
```

**Comment by djondi** [comment]
Hahahaha so simple yet caused so many problems. I've been Sublime Text fan for a year or two (before that used Dreamweaver), but the thing that bothered me about Sublime Text is it's looks, I'm not sure how to put it, but it looks rather childish to me. Environment is very important for creating code, at least for me. All other text editors have flaws, Atom without an option to load last project on startup, UltraEdit with it's complexity (simplicity is better^^), Vim with DOS kinda look and so on... Spent good amount of hours in past days to find the perfect editor. Brackets is the best so far that I've found :)

Thank you very much for helping me, I really appreciate it.
